### PR TITLE
Release 1.2.2

### PR DESCRIPTION
**Pull request by oprypin**

### CODE REVIEW

File: docs/about/release-notes.md [hunk 1]
```diff
@@ -21,15 +21,38 @@ The current and past members of the MkDocs team.
 * [@d0ugal](https://github.com/d0ugal/)
 * [@waylan](https://github.com/waylan/)
```
Comment (ultrabug) [inline docs/about/release-notes.md:22]: @oprypin you might want to add us there?

**Comment by ultrabug** [comment]
I still need to figure out a way to make the translation check pass when apparently the PR source branch is not fully up to date with master

File: docs/about/release-notes.md (hunk 1 above)
Comment (oprypin) [reply docs/about/release-notes.md:22]: I noticed this but intentionally decided not to add "us". Let's see how this plays out longer term first.

**Comment by oprypin** [comment]
#2506 is the last change that I intend to merge, then in 1 day I plan to release.

File: docs/about/release-notes.md (hunk 1 above)
Comment (oprypin) [reply docs/about/release-notes.md:22]: So, things played out quite calmly :)

[Add latest members of MkDocs maintenance team](https://github.com/mkdocs/mkdocs/pull/2661)

**Pull request merged**